<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceRecognitionWindow</class>
 <widget class="QMainWindow" name="FaceRecognitionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Recognition</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="videoLayout">
      <item>
       <widget class="QLabel" name="video_label">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="statusLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="doorbell_layout">
        <item>
         <widget class="QLabel" name="status_doorbell_frame">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #e0e0e0; border: 2px solid #999999; border-radius: 50%;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status_doorbell_label">
          <property name="text">
           <string>초인종</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="buzzer_layout">
        <item>
         <widget class="QLabel" name="status_buzzer_frame">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #e0e0e0; border: 2px solid #999999; border-radius: 50%;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status_buzzer_label">
          <property name="text">
           <string>부저</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="led1_layout">
        <item>
         <widget class="QLabel" name="status_led1_frame">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #e0e0e0; border: 2px solid #999999; border-radius: 50%;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status_led1_label">
          <property name="text">
           <string>LED1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="led2_layout">
        <item>
         <widget class="QLabel" name="status_led2_frame">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #e0e0e0; border: 2px solid #999999; border-radius: 50%;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status_led2_label">
          <property name="text">
           <string>LED2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="led3_layout">
        <item>
         <widget class="QLabel" name="status_led3_frame">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #e0e0e0; border: 2px solid #999999; border-radius: 50%;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status_led3_label">
          <property name="text">
           <string>LED3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="led4_layout">
        <item>
         <widget class="QLabel" name="status_led4_frame">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #e0e0e0; border: 2px solid #999999; border-radius: 50%;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status_led4_label">
          <property name="text">
           <string>LED4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>Start Recognition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_button">
          <property name="text">
           <string>Stop Recognition</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
